sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -649,6 +649,24 @@ def test_radius_neighbors_boundary_handling():
         assert_array_equal(results[0], [0, 1])
 
 
+def test_radius_neighbors_returns_array_of_objects():
+    """regression for issue #16036
+    """
```

Comment:
```suggestion
```

Don't use triple cote in test. It will overwrite the name of the function when executing pytest if I remember correctly.